🐛 BUG: tsconfig.json errors in an all-JS example project.

### Which Cloudflare product(s) does this pertain to?

Wrangler

### What version of `Wrangler` are you using?

3.1.1

### What operating system are you using?

macOS

### Describe the Bug

I ran `wrangler init`, and chose several options:

- `Where do you want to create your application?` 
cloudflare

- `What type of application do you want to create?`
Example router & proxy Worker

- `Do you want to use TypeScript?` 
No

Subsequently, cd'ing into `cloudflare` and running `npm run start` or `wrangler dev` near instantly returns:

```
occupymars@sourfruit cloudflare % npx wrangler dev
 ⛅️ wrangler 3.1.1
------------------
wrangler dev now uses local mode by default, powered by 🔥 Miniflare and 👷 workerd.
To run an edge preview session for your Worker, use wrangler dev --remote
✘ [ERROR] Expected string in JSON but found "compilerOptions"

    ../../../../tsconfig.json:1:1:
      1 │ {compilerOptions:{target:esnext, watch: true, lib: [dom, es2017]}}
        ╵  ~~~~~~~~~~~~~~~


✘ [ERROR] Expected string in JSON but found "compilerOptions"

    ../../../../tsconfig.json:1:1:
      1 │ {compilerOptions:{target:esnext, watch: true, lib: [dom, es2017]}}
        ╵  ~~~~~~~~~~~~~~~


✘ [ERROR] Build failed with 1 error:

  ../../../../tsconfig.json:1:1: ERROR: Expected string in JSON but found "compilerOptions"
  ```
  
  No tsconfig.json is anywhere to be found, and this is the default project...
  
  Cheers!

--- a/.changeset/sour-numbers-sing.md
+++ b/.changeset/sour-numbers-sing.md
@@ -1,0 +1,5 @@
+---
+"create-cloudflare": patch
+---
+
+Removes all typescript dependencies from javascript templates.

--- a/packages/create-cloudflare/templates/common/js/package.json
+++ b/packages/create-cloudflare/templates/common/js/package.json
@@ -7,7 +7,6 @@
 		"start": "wrangler dev"
 	},
 	"devDependencies": {
-		"@cloudflare/workers-types": "^4.20230419.0",
 		"itty-router": "^3.0.12",
 		"wrangler": "^3.0.0"
 	}

--- a/packages/create-cloudflare/templates/hello-world/js/package.json
+++ b/packages/create-cloudflare/templates/hello-world/js/package.json
@@ -7,8 +7,6 @@
 		"start": "wrangler dev"
 	},
 	"devDependencies": {
-		"@cloudflare/workers-types": "^4.20230419.0",
-		"typescript": "^5.0.4",
 		"wrangler": "^3.0.0"
 	}
 }

--- a/packages/create-cloudflare/templates/queues/js/package.json
+++ b/packages/create-cloudflare/templates/queues/js/package.json
@@ -7,8 +7,6 @@
 		"start": "wrangler dev"
 	},
 	"devDependencies": {
-		"@cloudflare/workers-types": "^4.20230419.0",
-		"typescript": "^5.0.4",
 		"wrangler": "^3.0.0"
 	}
 }

--- a/packages/create-cloudflare/templates/scheduled/js/package.json
+++ b/packages/create-cloudflare/templates/scheduled/js/package.json
@@ -7,8 +7,6 @@
 		"start": "wrangler dev"
 	},
 	"devDependencies": {
-		"@cloudflare/workers-types": "^4.20230419.0",
-		"typescript": "^5.0.4",
 		"wrangler": "^3.0.0"
 	}
 }

